src/plugins/index.ts

```diff
@@ -77,8 +77,8 @@ export const corePluginRegistry = new Registry<CorePluginConstructor>()
   .add("borders", BordersPlugin)
   .add("conditional formatting", ConditionalFormatPlugin)
   .add("figures", FigurePlugin)
-  .add("chart", ChartPlugin)
   .add("carousel", CarouselPlugin)
+  .add("chart", ChartPlugin)
```

Comment:
I think this is wrong,  carousel should know about charts and not the other way around.